### DISCUSSION TITLE
2022/02/15/%E6%96%87%E4%BB%B6%E5%82%A8%E5%AD%98/

### DISCUSSION THREAD

**Discussion by giscus** [duskdusts/duskdusts.github.io#3] [Q&A]
# 2022/02/15/%E6%96%87%E4%BB%B6%E5%82%A8%E5%AD%98/

首先是个人的文件大致分类 其中：  热文件由本地及onedrive personal 、阿里云盘、小米云服务存储 websaver数据非敏感且占用较大，全部放置云端，阿里云盘及和彩云具有全备份，onedrive business(edu)同步占用量较小的书籍及工程源文件，两份onedrive business(白嫖)同步视听部分 热文件定期提出一部分到websaver  存储位置分析：  阿里云盘

https://duskdust.top/2022/02/15/%E6%96%87%E4%BB%B6%E5%82%A8%E5%AD%98/

**Comment by duskdusts** [comment]
tset
